src/current/v25.1/understand-hotspots.md

```diff
@@ -0,0 +1,313 @@
+
+<img src="{{ 'images/v25.1/hotspots-figure-3.png' | relative_url }}" alt="incrementing INT primary key" style="border:1px solid #eee;max-width:100%" />
+
+If you are able to control for the performance degradation that occurs in Range 4 in this case, you do not change the fact that the system is now limited by how many writes a single range can execute. This then, in theory, limits CockroachDB to the performance of a single node, which is, by nature, antithetical to the goals of a distributed database.
+
+Index hotspots themselves are [hot by write](#write-hotspot), [moving hotspots](#moving-hotspot) which will continue to bounce around from range to range as the data grows.
+
+In the ideal operation of a distributed SQL database, inserts into an index should be distributed evenly in the index's span to avoid this type of hotspot. Each range responsible for storing keys receives an even share of the workload. If the ranges containing the data become hot in this mode, scaling the cluster is as easy as adding machines horizontally.
+
+Consider a table `users` which contains a primary keys `user_uuid` of type [`UUID`]({% link {{ page.version.version }}/uuid.md %}). Because `UUID`s are pseudo-random, new rows are inserted into the keyspace at random locations. The following image visualizes writes to the `users` table using a `UUID` primary key. Red lines indicate an insert into the keyspace. Note how the red lines are distributed evenly.
+
+<img src="{{ 'images/v25.1/hotspots-figure-2.png' | relative_url }}" alt="UUID primary key" style="border:1px solid #eee;max-width:100%" />
+
+Inserts are not the only way that index hotspots can occur. Consider the same table `users` which now has a [secondary index]({% link {{ page.version.version }}/schema-design-indexes.md %}) on a `TIMESTAMP` column:
+
+~~~ sql
+# ddl
+CREATE TABLE profiles(
+   user_uuid UUID PRIMARY KEY,
+   last_seen_at TIMESTAMP,
+   INDEX index_last_seen_at (last_seen_at)
+);
+
+# dml
+INSERT INTO profiles (user_uuid, last_seen_at) VALUES ('d9efa555-e8fd-4793-9239-f484454980cc', CURRENT_TIMESTAMP()), ('e3fabd81-2e08-4f25-99fd-8cd7e2928bd1', CURRENT_TIMESTAMP());
+UPDATE profiles SET last_seen_at = CURRENT_TIMESTAMP() WHERE user_uuid='d9efa555-e8fd-4793-9239-f484454980cc';
+~~~
+
+Because [`CURRENT_TIMESTAMP()`]({% link {{ page.version.version }}/functions-and-operators.md %}#date-and-time-functions) is a steadily increasing value, this [`UPDATE`]({% link {{ page.version.version }}/update.md %}) will similarly burden the range at the tail of the index. While hotspots on an index tail tend to be the most common, bottlenecks on the head are not unheard of. For example, indexing on `DESC` with the same insertion strategy will cause a hotspot.
```

Comment:
Could you rephrase "are not unheard of."? It could be a bit casual and also hard to localize. Maybe "are also possible."?